File: day23-hex/hex.js
var hex = "0xAB";

var hex_array = hex.split("");
hex_array.shift();
hex_array.shift();

hex_array.forEach(function(elem){
  if (elem == "A") {
    elem = "atta";
  } else if (elem == "B") {
    elem = "bibbity";
  } else if (elem == "C") {
    elem = "city";
  } else if (elem == "D") {
    elem = "dickety";
  } else if (elem == "E") {
    elem = "ebbity";
  } else if (elem == "F") {
    elem = "fleventy";
  }
});

hex_result = hex_array.join("-");

File: day23-hex/hex.rb
require "pry"

hex = "0xAB"

def pronounce(hex)
  hex_array = hex.split("").pop(2)
  binding.pry
  hex_array.each do |elem|
    if elem == "A"
      elem = "atta"
    elsif elem == "B"
      elem = "bibbity"
    elsif elem == "C"
      elem = "city"
    elsif elem == "D"
      elem = "dickety"
    elsif elem == "E"
      elem = "ebbity"
    elsif elem == "F"
      elem = "fleventy"
    end
  end
  hex_array
end

puts pronounce(hex)
puts "worked"
